tests/integration/api/controllers/replication.spec.js

```diff
@@ -833,7 +1067,7 @@ describe('replication', () => {
       expect(response.doc_ids).to.have.members(deletedIds);
     });
 
-    it('should return purged docs', async () => {
+    xit('should return purged docs', async () => {
```

Comment:
Same. All these tests are already working, and no extra changes are required. [I enabled them in a branch](https://github.com/medic/cht-core/actions/runs/9175810540).